### PR TITLE
Replace `` with html tag

### PR DESCRIPTION
**Pull request by JinsYin**
> https://jekyllrb.com/docs/variables/

### CODE REVIEW

**Comment by mattr-** [comment]
The motivation behind this change isn't clear. Can you please explain why it was necessary to make this change and what the benefit of this is?

File: docs/_data/jekyll_variables.yml [hunk 1]
```diff
@@ -51,10 +51,10 @@ site:
       <code>name</code>, <code>basename</code> and <code>extname</code>.
   - name: site.html_pages
     description: >-
-      A subset of `site.pages` listing those which end in `.html`.
+      A subset of <code>site.pages</code> listing those which end in <code>.html</code>.
   - name: site.html_files
     description: >-
-      A subset of `site.static_files` listing those which end in `.html`.
+      A subset of <code> site.static_files</code> listing those which end in <code>.html</code.
```
Comment (DirtyF) [inline docs/_data/jekyll_variables.yml]: variables-error

This change introduce a rendering problem, see https://deploy-preview-7879--jekyllrb.netlify.com/docs/variables/

Comment (JinsYin) [reply docs/_data/jekyll_variables.yml]: I have missed one `>` character

**Comment by DirtyF** [comment]
@jekyllbot: merge +docs

**Pull request merged**